<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedTab</class>
 <widget class="QWidget" name="AdvancedTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QGroupBox" name="soundGroup">
     <property name="title">
      <string>Sound</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="procButtonBox">
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="startupGroup">
     <property name="title">
      <string>Startup</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="saveGroup">
     <property name="title">
      <string>Save-related</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="fpsGroup">
     <property name="title">
      <string>FPS</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
